node/src/main/scala/coop/rchain/node/GetAllBalances.scala

```diff
@@ -0,0 +1,59 @@
+package coop.rchain.node
+
+import cats.Parallel
+import cats.effect.{Concurrent, ContextShift}
+import cats.implicits._
+import com.google.protobuf.ByteString
+import coop.rchain.casper.storage.RNodeKeyValueStoreManager
+import coop.rchain.casper.storage.RNodeKeyValueStoreManager.legacyRSpacePathPrefix
+import coop.rchain.metrics.{Metrics, NoopSpan}
+import coop.rchain.models.{BindPattern, ListParWithRandom, Par, TaggedContinuation}
+import coop.rchain.rholang.interpreter.RhoRuntime
+import coop.rchain.rspace.{Blake2b256Hash, Match, RSpace}
+import coop.rchain.shared.Log
+import coop.rchain.rspace.syntax._
+import coop.rchain.casper.util.rholang.VaultBalanceGetter
+import coop.rchain.crypto.codec.Base16
+
+import java.io._
+import java.nio.file.{Files, Path}
+import scala.concurrent.ExecutionContext
+
+object GetAllBalances {
+
+  def main[F[_]: Concurrent: Log: Parallel: ContextShift](
+      outputFile: Path,
+      stateHash: String,
+      dataDir: Path
+  )(implicit scheduler: ExecutionContext): F[Unit] = {
+    val oldRSpacePath = dataDir.resolve(s"$legacyRSpacePathPrefix/history/data.mdb")
```

Comment:
What is the meaning of `$` at the beginning of the path string?

Reply:
It is the same as "${legacyRSpacePathPrefix}/history/data.mdb"

Reply:
:)